app/components/ObservationCard.tsx

```diff
@@ -0,0 +1,131 @@
+import { Card, Elevation, Tooltip } from '@blueprintjs/core';
+import classNames from 'classnames/bind';
+import path from 'path';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import CreatableSelect from 'react-select/creatable';
+
+import { formatAnimalClassName } from '../constants/animalsClasses';
+import { taxonOptions } from '../constants/taxons';
+import styles from './ObservationCard.module.scss';
+
+const cx = classNames.bind(styles);
+
+type ObservationCardProps = {
+  observation: Observation;
+  predictionOverride?: CreatableOption;
+  onPredictionOverride: PredictionOverrideHandler;
+  onPhotoClick: (cardIndex: number | null) => void;
+  observationIndex: number;
+  isMaximized: boolean;
+};
+
+export default function ObservationCard(props: ObservationCardProps) {
+  const {
+    observation,
+    predictionOverride,
+    onPredictionOverride,
+    onPhotoClick,
+    observationIndex,
+    isMaximized
+  } = props;
+  const { t } = useTranslation();
+  const [isOverriden, setOverride] = useState<boolean>(false);
+
+  const dropdownInitialValue = {
+    value: observation.pred_1,
+    label: formatAnimalClassName(observation.pred_1)
+  };
+
+  const handlePredictionOverride = (newValue: CreatableOption | null) => {
+    onPredictionOverride(observation.location, newValue);
+    setOverride(
```

Comment:
I've added this back from previous commit, to fix react select element which didn't update correctly. Steps to reproduce:
1. Override some card
2. Using dropdown select back the original value of this overriden card
3. The override is gone but we still can see a cross icon to remove override

Reply:
Unfortunately this won't work, as the state will be lost when the card will be unrendered in the virtualized container: override a prediction, scroll photos to the bottom and back; the "clear" button will be lost.

I have pushed a commit which should resolve this.